### ISSUE TITLE
Stats Improvement: There's no focus highlight/style on the dual choice popover

### ISSUE THREAD

**Issue by kangzj** [Automattic/wp-calypso#78217]
<img width="490" alt="image" src="https://github.com/Automattic/wp-calypso/assets/390760/1c815621-7fae-4ce6-8580-e94beba0f4bd">

**Comment by kangzj** [comment]
Fixed with #80408
